packages/wallets/src/utils/calculate-total-by-key.ts

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
@hasan-deriv 
ideally we are moving these utils to to https://github.com/deriv-com/deriv-utils, if it possible to do, it would be great

Reply:
this may delay the release of this feature as now we'd need to open another PR in deriv-utils, merge it, and then update the package here. If this util is only scoped to this feature we may keep it here